src/Apps/Order/Utils/getErrorDialogCopy.tsx

```diff
@@ -0,0 +1,80 @@
+import { RouterLink } from "System/Router/RouterLink"
+import React from "react"
+
+const SUPPORT_EMAIL = "orders@artsy.net"
+
+export enum ErrorDialogs {
+  CurrencyNotSupported = "currency_not_supported",
+}
+
+interface ErrorDialogCopy {
+  title: string
+  message: string
+  formattedMessage: React.ReactNode
+}
+
+export const getErrorDialogCopy = (dialog?: ErrorDialogs): ErrorDialogCopy => {
+  switch (dialog) {
+    case ErrorDialogs.CurrencyNotSupported:
+      return currencyNotSupportedErrorDialogCopy()
+    default:
+      return defaultErrorDialogCopy()
+  }
+}
+
+const currencyNotSupportedErrorDialogCopy = () => {
+  const title = "Payment declined"
+  const message = `This card is not compatible with the currency for this artwork. Please confirm with your card issuer if this currency can be supported, try a different payment method or contact ${SUPPORT_EMAIL}.`
```

Comment:
This looks good, but can we use our i18n helpers? https://github.com/artsy/force/blob/main/src/System/i18n/locales/en-US/translation.json

Reply:
(Also cool to punt on this and update everything all at once -- just want to be sure y'all are aware of this.)

Reply:
I'm sort of unclear on how those helpers plan to be actually translated in the future and vastly prefer having copy inline, vs. an additional layer of abstraction and naming in that JSON file. Have we decided one way or another on an enforced standard?

cc @dzucconi who i know was also opinionated

Reply:
Yah, it's not blocking or anything, but all that being said, we've made a lot of headway here. (Most) folks reach towards i18n by default. 

And with ChatGPT, having a centralized store of languages like this will be very helpful for other languages first-passes, at some future point in time.

Reply:
IMO putting things in that file at this point makes the codebase harder to navigate. There's no structure to it and it hasn't manifested in any actual benefits.

Reply:
(That said: I also find the idea of a function to return copy also confusing — seems like it'd be much more clear inline.)